framework/wazuh/core/common.py

```diff
@@ -134,13 +134,22 @@ def get_context_cache() -> dict:
 origin_module: ContextVar[str] = ContextVar('origin_module', default='framework')
 try:
     mp_pools: ContextVar[Dict] = ContextVar('mp_pools', default={
-        'process_pool': ProcessPoolExecutor(max_workers=1),
-        'events_pool': ProcessPoolExecutor(max_workers=1)
+        'process_pool': {
+            'pool': ProcessPoolExecutor(max_workers=1),
+            'size':1
+        },
+        'events_pool':  {
+            'pool': ProcessPoolExecutor(max_workers=1),
+            'size': 1
```

Comment:
Unify the values in a single constant to clarify that the values of `max_workers` and `size` should be the same.

Reply:
https://github.com/wazuh/wazuh/pull/28757/commits/7328b0dbb8215383ab8267e648b1e8145b2ccc7e